test/server/lib/db/query.js

```diff
@@ -212,4 +212,49 @@ describe('lib/db/query', () => {
 			);
 		});
 	});
+
+	describe('hourlyUsage', () => {
+		const stub = sinon.stub();
+		const { hourlyUsage } = proxyquire('../../../../src/lib/db/query', {
+			'./postgres': { manyOrNone: stub }
+		});
```

Comment:
Can this be moved to the top?

Reply:
I mean it _could_ but it's only used in this scope, so why bother polluting the parent scopes with it?

Reply:
Fixed